app/helpers/startup.js

```diff
@@ -12,11 +13,13 @@ module.exports = () => {
 
       appLauncher.isEnabled().then((enabled) => {
         if (enabled) return
+        logger.log('info', 'Adding to linux startup')
         return appLauncher.enable()
       })
     } else {
       const settings = app.getLoginItemSettings()
       if (!settings.openAtLogin) {
+        logger.log('info', 'Adding to win32 or darwin startup')
```

Comment:
actually they could be info if it was triggered by a user, checking is silly so keep it as info.